cmd/helm/plugin_install.go

```diff
@@ -40,7 +40,7 @@ func newPluginInstallCmd(out io.Writer) *cobra.Command {
 	o := &pluginInstallOptions{}
 	cmd := &cobra.Command{
 		Use:     "install [options] <path|url>...",
```

Comment:
If this is true, then this should remove the ellipsis.

Reply:
done